corehq/apps/translations/templates/convert_translations.html

```diff
@@ -0,0 +1,15 @@
+{% extends 'translations/base_template.html' %}
+{% load crispy_forms_tags %}
+{% load xforms_extras %}
+{% load url_extras %}
+{% load hq_shared_tags %}
+{% load i18n %}
+{% load compress %}
+{% load djangular_tags %}
```

Comment:
This isn't used...actually, most of these aren't used, the angular one just caught my eye because we're deprecating angular.

Reply:
Thanks, was meant to remove this earlier.